Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -95,11 +98,18 @@ props.buildReportOrder.each{ buildReportFile ->
 	// finds all the build outputs with a deployType
 	def buildRecords = buildReport.getRecords().findAll{
 		try {
-			(it.getType()==DefaultRecordFactory.TYPE_EXECUTE || it.getType()==DefaultRecordFactory.TYPE_COPY_TO_PDS) &&
-					!it.getOutputs().isEmpty()
+            (it.getType()==DefaultRecordFactory.TYPE_EXECUTE || it.getType()==DefaultRecordFactory.TYPE_COPY_TO_PDS) &&
+                  !it.getOutputs().isEmpty()
 		} catch (Exception e){}
 	}
 
+    // finds all the build outputs with a deployType
+    def USSBuildRecords = buildReport.getRecords().findAll{
```

Comment:
Use camel case instead of starting with upper case, which makes it look like the type of an object. See above

Reply:
Done